main.go

```diff
@@ -325,6 +331,57 @@ func (s *service) openNewIssues(issueMap map[string]*github.Issue, comments []*t
 	log.Printf("Created new issues. count=%v", count)
 }
 
+func (s *service) assignNewIssues(issueMap map[string]*github.Issue, assigneeMap *map[string]string) {
+	for title, assignee := range *assigneeMap {
+		issue := issueMap[title]
+		issueNumber := issue.GetNumber()
+		req := &github.IssueRequest{
+			Assignees: &[]string{assignee},
+		}
+		_, _, err := s.client.Issues.Edit(s.ctx, s.env.owner, s.env.repo, issueNumber, req)
+		if err != nil {
+			log.Printf("Error while assigning %v to issue %v. err=%v", assignee, issueNumber, err)
+		} else {
+			log.Printf("Successfully assigned %v to issue %v.", assignee, issueNumber)
+		}
+	}
+}
+
+func (s *service) getAssigneeFromCommitHash(commitHash string, title string, assigneeMap *map[string]string, successCount *int, assigneeMapMux sync.Locker, wg *sync.WaitGroup) {
```

Comment:
1. Usually method `getXYZ()` returns something. This method does not return anything. I propose to rename it to `retrieveCommitAssignees()` or something along these lines.
2. There's no need to store `successCount` as the size of `assigneeMap` is equal to `successCount`

Reply:
Regarding point 2, `successCount` is not incremented if there is an error getting the commit author while calling the github API. So, not always the size of `assigneeMap`.

Reply:
Okay, I understand what you mean now 😛 
Fixing this.